base.cfg

```diff
@@ -65,6 +66,7 @@ recipe = zc.recipe.testrunner
 eggs = ${instance:eggs}
 initialization =
     os.environ['TZ'] = 'UTC'
+    os.environ['ZSERVER_PORT'] = '55001'
```

Comment:
I see the need for a stable port in the documentation HTTP examples, but doesn't this cause issues with parallel runs on Jenkins when the Jenkins port allocator plugin isn't in full control of handing out ports?

Reply:
@lukasgraf I also thought about it but then I realised that the coredev has its own buildout. Or am I wrong?

Reply:
@sneridagh I think this is already set in .travis.yml.

Reply:
ok, we need this locally as well of course. Ignore my previous comment.